build.gradle

```
@@ -30,6 +30,8 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-validation'
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.boot:spring-boot-starter-security'
+    implementation platform('software.amazon.awssdk:bom:2.21.32')
+    implementation 'software.amazon.awssdk:s3'
```

jwt 밑으로 옮겨주시고 s3 주석 추가해주세요


```suggestion
    // s3
    implementation platform('software.amazon.awssdk:bom:2.21.32')
    implementation 'software.amazon.awssdk:s3'
```